Framework/Algorithms/test/ReflectometryBackgroundSubtractionTest.h

```diff
@@ -0,0 +1,124 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_ALGORITHMS_REFLECTOMETRYBACKGROUNDSUBTRACTIONTEST_H_
+#define MANTID_ALGORITHMS_REFLECTOMETRYBACKGROUNDSUBTRACTIONTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidAlgorithms/ReflectometryBackgroundSubtraction.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+using Mantid::Algorithms::ReflectometryBackgroundSubtraction;
+using namespace Mantid::API;
+
+class ReflectometryBackgroundSubtractionTest : public CxxTest::TestSuite {
+private:
+  MatrixWorkspace_sptr m_multiDetectorWSWithPeak;
+
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static ReflectometryBackgroundSubtractionTest *createSuite() {
+    return new ReflectometryBackgroundSubtractionTest();
+  }
+  static void destroySuite(ReflectometryBackgroundSubtractionTest *suite) {
+    delete suite;
+  }
+
+  ReflectometryBackgroundSubtractionTest() {
+    FrameworkManager::Instance();
+    // A multi detector ws
+    m_multiDetectorWSWithPeak = WorkspaceCreationHelper::
+        create2DWorkspaceWithReflectometryInstrumentMultiDetector();
```

Comment:
The name suggests this workspace has a peak, but I don't think it does? It would be a better test if it did have a peak.